src/MonteCarloMoves/MonteCarloMoves.jl

```diff
@@ -0,0 +1,38 @@
+"""
+Module containing functions for performing Monte Carlo moves on atomic/molecular systems.
+"""
+module MonteCarloMoves
+
+using ExtXYZ
+using AtomsBase
+using Setfield
+using Distributions
+using Unitful
+using StaticArrays
+# using LinearAlgebra
+
+using ..Potentials
+using ..Hamiltonians
+using ..AbstractWalkers
+using ..EnergyEval
+
+export periodic_boundary_wrap!
+export MC_random_walk!, MC_nve_walk!, MC_new_sample!
+export generate_random_new_lattice_sample!
+
+include("helpers.jl")
+
+include("random_walks.jl")
+
+include("nve_walks.jl")
+
+
```

Comment:
@yangmr04, should we remove lines 29–37?

Reply:
Done!